lib/library.go

```diff
@@ -93,6 +93,41 @@ func ExtractIdentityFromContactCode(bundleString *C.char) *C.char {
 	return C.CString(string(data))
 }
 
+// VerifySignatures ensure the signature pairs is valid
+//export VerifySignatures
+func VerifySignatures(signaturePairsStr *C.char) *C.char {
+	var signaturePairs [][]string
```

Comment:
it is not possible to unmarshal it into slice of byte arrays? `[][3]byte]`

Reply:
I take from your comment it is possible and a good way :), so will do that, still a go newbie here, thanks!